docs/content/en/docs/Developers/factorycube-edge/cameraconnect/index.md

```diff
@@ -2,31 +2,95 @@
 title: "cameraconnect"
 linkTitle: "cameraconnect"
 description: >
-  This docker container automatically detects cameras in the network and makes them accessible via MQTT. The MQTT output is specified in [the MQTT documentation](/docs/concepts/mqtt/)
----
+  This docker container automatically detects cameras in the network and makes them accessible via MQTT. 
+  The MQTT output is specified in [the MQTT documentation](/docs/content/en/docs/Concepts/mqtt)
+
 
 **This microservice is still in development and is not considered stable for production use.**
 
 ## Getting started
 
 ### Using the Helm chart
 
-By default cameraconnect will be deactivated in factorycube-edge. First, you need to enable it in the factorycube-edge values. Then you need to create a folder on the node in `/home/rancher/gentl_producer` and move your genTL producer files (*.cti) and all required libaries into that folder. Then apply your settings to the Helm chart with `helm upgrade`.
+By default cameraconnect will be deactivated in factorycube-edge. First, 
+you need to enable it in the factorycube-edge values. 
+Then you need to create a folder on the node in `/home/rancher/genticam_producer`
+and move your genTL producer files (*.cti) and all required libaries into that folder. 
+Then apply your settings to the Helm chart with `helm upgrade`.
 
-Another idea for changing it:
-`helm install ....... --set 'cameraconnect.enabled=true'`
+With Lens you can change the values under the apps/releases tab on the left side. 
```

Comment:
you could refer here to the tutorial https://github.com/united-manufacturing-hub/united-manufacturing-hub/blob/staging/docs/content/en/docs/Tutorials/united-manufacturing-hub/working-with-helm-and-lens/index.md

It is far from complete, but we avoid double content